### PR TITLE
Dev 1551: Use Secure Exchange with Plaid

### PR DESCRIPTION
**Pull request by ShreyaThapa**
- Includes changes from DEV-1539: Restructure project
- Update Plaid example app to use Secure Exchange

### CODE REVIEW

File: packages/secure-token-exchange/plaid/src/pages/create-customers.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+    /**
+     * Mutates the form state if the input value changes.
+     */
+    function handleInputChanged({ target: { name, value } }: ChangeEvent<HTMLInputElement>) {
+        setFormData({ ...formData, [name]: value });
+    }
+
+    return (
+        <MainLayout title="Step 1: Create Customers">
```
Comment (spencerhunter) [inline packages/secure-token-exchange/plaid/src/pages/create-customers.tsx:91]: Small change: Update to `Step 1: Create Customer` (remove s)

File: packages/secure-token-exchange/plaid/src/pages/create-customers.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+            {alert && (
+                <Alert severity={alert.severity} sx={{ mb: 2 }}>
+                    {alert.message}
+                </Alert>
+            )}
+
+            <Card sx={{ padding: 3 }}>
+                <CardHeader title="Create Customers" />
```
Comment (spencerhunter) [inline packages/secure-token-exchange/plaid/src/pages/create-customers.tsx:99]: same as above

File: packages/secure-token-exchange/plaid/src/pages/create-customers.tsx (hunk 1 above)
Comment (spencerhunter) [inline packages/secure-token-exchange/plaid/src/pages/create-customers.tsx:91]:
```suggestion
        <MainLayout title="Step 1: Create Customer">
```

File: packages/secure-token-exchange/plaid/src/pages/create-customers.tsx (hunk 2 above)
Comment (spencerhunter) [inline packages/secure-token-exchange/plaid/src/pages/create-customers.tsx:99]:
```suggestion
                <CardHeader title="Create Customer" />
```

File: packages/secure-token-exchange/plaid/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Using Plaid Link
+
+When using [Plaid Link](https://plaid.com/docs/link/#introduction-to-link) to connect a bank account, use the following credentials to successfully verify a bank.
+
+```bash
+username: user_good
+password: pass_good
+```
+
+To simulate different flows, you can use different test credentials that can be found in [Plaid's website](https://plaid.com/docs/sandbox/test-credentials/).
```
Comment (spencerhunter) [inline packages/secure-token-exchange/plaid/README.md:25]:
```suggestion
To simulate different flows, you can use different test credentials that can be found on [Plaid's website](https://plaid.com/docs/sandbox/test-credentials/).
```

File: packages/secure-token-exchange/plaid/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# Dwolla and Plaid - Secure Exchange
+
+This example application, built using [Next.js](https://nextjs.org/), demonstrates how a Funding Source can be created for a Dwolla Customer using Dwolla's integration with Plaid via Dwolla's [Secure Exchange](https://developers.dwolla.com/docs/balance/secure-exchange) solution. By doing this, Dwolla is able to instantly verify the Funding Source without the need for your application to transmit sensitive data. (All sensitive data is retrieved directly from MX by Dwolla.)
```
Comment (spencerhunter) [inline packages/secure-token-exchange/plaid/README.md:3]:
```suggestion
This example application, built using [Next.js](https://nextjs.org/), demonstrates how a Funding Source can be created for a Dwolla Customer using Dwolla's integration with Plaid via Dwolla's [Secure Exchange](https://developers.dwolla.com/docs/balance/secure-exchange) solution. By doing this, Dwolla is able to instantly verify the Funding Source without the need for your application to transmit sensitive data. (All sensitive data is retrieved directly from Plaid by Dwolla.)
```

**Review by spencerhunter — CHANGES_REQUESTED** [review]

File: packages/secure-token-exchange/plaid/README.md (hunk 4 above)
Comment (ShreyaThapa) [reply packages/secure-token-exchange/plaid/README.md:3]: Ope! Thanks for catching that!

**Review by spencerhunter — APPROVED** [review]

**Pull request merged**